test/e2e/framework/util.go

```diff
@@ -3509,7 +3509,7 @@ func AllNodesReady(c clientset.Interface, timeout time.Duration) error {
 		// It should be OK to list unschedulable Nodes here.
 		nodes, err := c.Core().Nodes().List(metav1.ListOptions{})
 		if err != nil {
-			return false, err
+			return false, nil
```

Comment:
Depending on the error. If the error is non-retryable one, I think that we actually should return an error here.